Description: If GRUB Legacy is still around, tell packaging to ignore it
Author: Colin Watson <cjwatson@debian.org>
Bug-Debian: http://bugs.debian.org/586143
Forwarded: not-needed
Last-Update: 2010-06-18

Index: b/util/grub-install.in
===================================================================
--- a/util/grub-install.in
+++ b/util/grub-install.in
@@ -863,6 +863,13 @@
     echo 1>&2
 fi
 
+# If vestiges of GRUB Legacy still exist, tell the Debian packaging that
+# they can ignore them.
+if test -z "$rootdir" && \
+   test -e /boot/grub/stage2 && test -e /boot/grub/menu.lst; then
+    touch /boot/grub/grub2-installed
+fi
+
 gettext "Installation finished. No error reported." 1>&2
 echo 1>&2
 
